packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -3978,6 +3981,20 @@ class ExcludeSemantics extends SingleChildRenderObjectWidget {
   }
 }
 
+class _ExcludeSemanticsElement extends SingleChildRenderObjectElement {
+  _ExcludeSemanticsElement(ExcludeSemantics widget) : super(widget);
+
+  @override
+  ExcludeSemantics get widget => super.widget;
+
+  @override
+  void visitChildrenForSemantics(ElementVisitor visitor) {
+    if (widget.excluding)
+      return;
```

Comment:
That's how `ExcludeSemantics` works at the `RenderObject` level too. It excludes the entire sub-tree.